README.md

```diff
@@ -14,6 +14,12 @@ docker pull archiveteam/warrior-dockerfile
 # use -p to bind port 8001 on the docker container 
 # (default ip 172.17.0.x) to port 8001 on localhost.
 docker run [-d] [-p 8001:8001] archiveteam/warrior-dockerfile
+
+# Restart on reboot automatically by just running:
+docker run --detach \
+  --publish 8001:8001 \
+  --restart always \
+  archiveteam/warrior-dockerfile
```

Comment:
Doing `docker run` will create new instance of the container right? Having this command right after the previous one in the same code block might be confusing if people just copy and paste. IE, `run` vs `start` might not be obvious to new users.

Maybe add instructions that says to shutdown the warrior, reconfigure the restart policy, and then run it again in the background daemon?